pontoon/contributors/templates/contributors/widgets/notifications_menu.html

```diff
@@ -1,14 +1,19 @@
 {% macro menu() %}
 {% if user.is_authenticated() %}
+
 {% set unread_notifications = user.notifications.unread() %}
-<div id="notifications" class="notifications select{% if unread_notifications|length > 0 %} unread{% endif %}">
+{% set unread_notifications_count = unread_notifications.count() %}
+{% set limit = 7 %}
+{% set notifications_to_display_count = unread_notifications_count if unread_notifications_count > limit else limit %}
+
+<div id="notifications" class="notifications select{% if unread_notifications_count > 0 %} unread{% endif %}">
```

Comment:
I think you could remove that '> 0' part and do something like:  `{{ ' unread' if unread_notifications_count else '' }}`

Reply:
Updated (not exactly as you suggested though).